lib/compiler.js

```diff
@@ -50,8 +50,10 @@ exports.compile = function (code, options) {
     return result;
   }
 
-  const rootPath = FastPath.from(ast);
+  options.moduleId = options.moduleId ||
+    makeUniqueIdQuickly("module", code);
```

Comment:
It's possible to pass in `options.moduleId` explicitly, to override/skip calling `makeUniqueIdQuickly`.